parseable/benchmark.sh

```diff
@@ -0,0 +1,38 @@
+# Install Dependencies
+sudo apt-get update -y
+sudo apt-get install -y rustc rustup
+rustup update stable
+
+
+# Clone Parseable Repository and checkout v1.7.3 branch
+git clone https://github.com/parseablehq/parseable.git
+cd parseable
+git checkout v1.7.3
+
+# Build and Run Parseable
+cargo build --release
+export PATH="`pwd`/target/release:$PATH"
+export RUST_LOG=debug
+./target/release/parseable local-store > parseable.log 2>&1 & PARSEABLE_PID=$!
+# Verify Parseable is running
+if ps -p $PARSEABLE_PID > /dev/null; then
+    echo "Parseable is running with PID: $PARSEABLE_PID"
+else
+    echo "Error: Parseable failed to start. Check parseable.log for details."
+    exit 1
+fi
+
+#run ingestion script
+./ingestion.sh
+
+#sleep for 3 minutes to allow sync to complete
+sleep 180
+
+#run query script
+./run_query.sh
```

Comment:
When I ran ./run_query.sh (after ingesting the data), I got errors of this kind:

```
========================================
Query 31: SELECT "SearchEngineID", "ClientIP", COUNT(*) AS c, SUM("IsRefresh"), AVG("ResolutionWidth") FROM hits WHERE "SearchPhrase" <> '' GROUP BY "SearchEngineID", "ClientIP" ORDER BY c DESC LIMIT 10;
Iteration 1:
jq: parse error: Invalid numeric literal at line 1, column 10
Time: 0.082227547 seconds
----------------------------------------
Iteration 2:
jq: parse error: Invalid numeric literal at line 1, column 10
Time: 0.009822313 seconds
----------------------------------------
Iteration 3:
jq: parse error: Invalid numeric literal at line 1, column 10
Time: 0.010060168 seconds
----------------------------------------
```